Source/Shaders/PolylineCommon.glsl

```diff
@@ -32,30 +32,48 @@ void clipLineSegmentToNearPlane(
             clipped = true;
         }
     }
-    
+
     positionWC = czm_eyeToWindowCoordinates(vec4(p0, 1.0));
 }
 
-vec4 getPolylineWindowCoordinates(vec4 position, vec4 previous, vec4 next, float expandDirection, float width, bool usePrevious) {
+vec4 getPolylineWindowCoordinates(vec4 position, vec4 previous, vec4 next, float expandDirection, float width, bool usePrevious, out float angle) {
     vec4 endPointWC, p0, p1;
     bool culledByNearPlane, clipped;
-    
+
     vec4 positionEC = czm_modelViewRelativeToEye * position;
     vec4 prevEC = czm_modelViewRelativeToEye * previous;
     vec4 nextEC = czm_modelViewRelativeToEye * next;
-    
+
+    // Compute the window coordinates of the points.
+    vec4 positionWindow = czm_eyeToWindowCoordinates(positionEC);
+    vec4 previousWindow = czm_eyeToWindowCoordinates(prevEC);
+    vec4 nextWindow = czm_eyeToWindowCoordinates(nextEC);
+
+    // Determine the relative screen space direction of the line.
+    vec2 lineDir;
+    if (usePrevious) {
+        lineDir = normalize(positionWindow.xy - previousWindow.xy);
+    }
+    else {
+        lineDir = normalize(nextWindow.xy - positionWindow.xy);
+    }
+    angle = atan(lineDir.x, lineDir.y) - atan(1.0, 0.0);
```

Comment:
Last time I tested - which was awhile ago - atan is pretty much the worse thing we can do in a shader.  @bagnell said there was already inverse trig in the polyline shader, but I don't see it at quick glance.

If you're not up for it now, we'll try to make this change soon.